PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/List/VideoTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class VideoTableViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  var tableOptions : Array<PlayerSelectionOption>!
+  var tableView: UITableView!
+  
+  var rowHeight: CGFloat!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    tableView = UITableView(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: self.view.frame.height), style: .plain)
+    self.view.addSubview(tableView)
+    
+    self.tableView.dataSource = self
+    self.tableView.delegate = self
+    
+    self.rowHeight = 100
+    self.populateOptions()
+    
+  }
+  override func viewWillLayoutSubviews() {
+    super.viewWillLayoutSubviews()
+    tableView.frame = view.bounds
+  }
+  
+  func numberOfSectionsInTableView(tableView: UITableView) -> Int {
```

Comment:
We have 2 functions with similar signature:
```  func numberOfSectionsInTableView(tableView: UITableView) -> Int``` and
```func numberOfSections(in tableView: UITableView) -> Int```

but we can delete both of them because according to the docs:
```- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView;              // Default is 1 if not implemented```

Reply:
Done